frontend/packages/ux-editor/src/components/config/RuleComponent.tsx

```diff
@@ -109,29 +111,29 @@ class Rule extends React.Component<IRuleComponentProps, any> {
     const selectedMethod = this.state.ruleConnection.selectedFunction;
     const selectedMethodNr = this.state.selectedFunctionNr;
     return (
-      <div className='modal-content'>
-        <div className='modal-header a-modal-header'>
-          <div className='a-iconText a-iconText-background a-iconText-large'>
-            <div className='a-iconText-icon'>
-              <i className='fa fa-corp a-icon' />
-            </div>
-            <h1 className='a-iconText-text mb-0'>
-              <span className='a-iconText-text-large'>
-                {this.props.t('ux_editor.modal_configure_rules_header')}
-              </span>
-            </h1>
-          </div>
+      <Modal
+        isOpen={true}
+        onRequestClose={() => {}}
```

Comment:
We should pass the cancelEdit callback to the onRequestClose to ensure that the dialog closes when the user clicks on the backdrop.
```suggestion
        onRequestClose={this.props.cancelEdit}
```